sys/include/net/gnrc/gnrc_mac/gnrc_mac.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_mac  A common MAC type for providing key MAC parameters and helper functions
+ * @ingroup     net
+ * @brief       A common MAC type for providing key MAC parameters and helper functions.
+ *              
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the GNRC_MAC
+ *
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ * @author      Shuguo Zhuo  <shuguo.zhuo@inria.fr>
+ */
+
+#ifndef GNRC_MAC_H
+#define GNRC_MAC_H
+
+#include <kernel_types.h>
+#include <net/gnrc/netdev2.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Max link layer address length in bytes */
+#ifndef GNRC_MAC_MAX_L2_ADDR_LEN
+#define GNRC_MAC_MAX_L2_ADDR_LEN           (8U)
```

Comment:
This again is a value [we already have defined](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/gnrc/netif/hdr.h#L38). I know that file might not be the most ideal location for that, so we might want to move it, but a new `define` is not really beneficial.